api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCcrCohortQueries.java

```diff
@@ -0,0 +1,3014 @@
+   *       FR7)
+   *   <li>Filtrando as que tiveram o registo de “Tratamento Nutricional ATPU” igual a "Sim” na
+   *       primeira “Ficha de Seguimento de CCR” registada durante o período de reporte (“Data da
+   *       Consulta” >= “Data Início” e <= “Data Fim”)..
+   * </ul>
+   *
+   * <p><b>Mota:</b> em caso de existirem mais que uma “Ficha de Seguimento de CCR” durante o
+   * período será considerada a informação registada na primeira ficha.
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition getChildrenWhoReceivedNutritionalTreatment() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.setName("Crianças que receberam ATPU");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Health Facility", Location.class));
+
+    cd.addSearch("firstConsultation", map(getPatients1stConsultation(), mapping));
+    cd.addSearch("receivedAtpu", map(getChildrenWhoReceivedAtpu(), mapping));
+
+    cd.setCompositionString("firstConsultation AND receivedAtpu");
+    return cd;
+  }
+
+  /**
+   * CCR-FR15
+   *
+   * <p><b>Indicador 9 - </b>Crianças que receberam CSB/suplemento nutricional
+   *
+   * <p>O sistema irá produzir o Indicador 9 “Total de crianças que receberam CSB/suplemento
+   * nutricional”, da seguinte forma:
+   *
+   * <ul>
+   *   <li>Incluindo todas as crianças que tiveram a 1ª consulta durante o período de reporte (CCR-
+   *       FR7)
+   *   <li>Filtrando as que tiveram o registo de “Tratamento Nutricional CSB” igual a "Sim” na
+   *       primeira “Ficha de Seguimento de CCR” registada durante o período de reporte (“Data da
+   *       Consulta” >= “Data Início” e <= “Data Fim”).
+   * </ul>
+   *
+   * <p><b>Mota:</b> em caso de existirem mais que uma “Ficha de Seguimento de CCR” durante o
+   * período será considerada a informação registada na primeira ficha.
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition getChildrenWhoReceivedCsbOrNutritionalSuplement() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.setName("Crianças que receberam CSB/suplemento nutricional");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Health Facility", Location.class));
+
+    cd.addSearch("firstConsultation", map(getPatients1stConsultation(), mapping));
+    cd.addSearch("receivedCsb", map(getChildrenWhoReceivedCsb(), mapping));
+
+    cd.setCompositionString("firstConsultation AND receivedCsb");
+    return cd;
+  }
+
+  public CohortDefinition getInfantAge(boolean greaterThan, Integer Age) {
+    SqlCohortDefinition sqlCohortDefinition = new SqlCohortDefinition();
```

Comment:
Please follow the camelCase pattern to define variables

Reply:
Resolved